### PR TITLE
Feature/kajal

### PR DESCRIPTION
**Pull request by kkaajjaall**
file added

### CODE REVIEW

**Review by ithqindia — COMMENTED** [review]
Correct spelling is **permanent**

File: forms.blade.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    {{--  permananat address form --}}
+    <div class="container permanant_address">
+        <form class="row g-3 ">
+            <h4> Address(permanant)</h4>
+            <div class="col-12">
+                <label for="address" class="form-label">Address</label>
+                <input type="text" class="form-control" id="permanant_address" name="permanant_address">
+            </div>
+            <div class="col-12">
+                <label for="apt" class="form-label">Apt/Suit/etc</label>
```
Comment (ithqindia) [inline forms.blade.php]: Apartment, Suite, etc

File: forms.blade.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <input type="text" class="form-control" id="permanant_apt" name="permanant_apt">
```
Comment (ithqindia) [inline forms.blade.php]: permanent_apartment

File: forms.blade.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
+@extends('layouts.main')
+@section('home', 'form-page')
+@section('container')
+<section>
+    {{--  permananat address form --}}
+    <div class="container permanant_address">
+        <form class="row g-3 ">
+            <h4> Address(permanant)</h4>
```
Comment (ithqindia) [inline forms.blade.php]: Permanent address

File: forms.blade.php [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                <input type="text" class="form-control" id="permanant_apt" name="permanant_apt">
+            </div>
+            <div class="col-12">
+                <label for="town" class="form-label">City/Town</label>
+                <input type="text" class="form-control" id="permanant_town" name="permanant_town" required>
+            </div>
+            <div class="col-12">
+                <label for="pin" class="form-label">Pincode</label>
+                <input type="number" class="form-control" id="permanant_pin" name="permanant_pin"
+                    placeholder="****" required>
```
Comment (ithqindia) [inline forms.blade.php]: Remove placeholder

File: forms.blade.php [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+            </div>
+            <div class="col-12">
+                <label for="state" class="form-label">State</label>
+                <input type="text" class="form-control" id="permanant_state" name="permanant_state" required>
+            </div>
+            <div class="col-12">
+                <label for="country" class="form-label">country</label>
```
Comment (ithqindia) [inline forms.blade.php]: `<label for="country" class="form-label">Country</label>`
Capital C

File: forms.blade.php [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+                <input type="text" class="form-control" id="permanant_country" name="permanant_country" required>
+            </div>
+            <div class="col-12">
+                <button type="submit" class="btn btn-primary" id=permanant_submit> Submit</button>
```
Comment (ithqindia) [inline forms.blade.php]: Missing quotation mark in id

File: forms.blade.php [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            </div>
+        </form>
+    </div>
+     {{--  local address form --}}
+    <div class="container mt-3 local_address">
+        <h4> Local Address</h4>
+        <div class="form-check mt-5">
+            <input class="form-check-input rbtn" type="radio" name="radiobtn" id="same"
```
Comment (ithqindia) [inline forms.blade.php]: name should be meaningful

File: forms.blade.php [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+                value="Same as above">
+            <label class="form-check-label" for="flexRadioDefault2">
+                Same as above
+            </label>
+        </div>
+        <div class="form-check mt-2">
+            <input class="form-check-input rbtn" type="radio" name="radiobtn" id="new" value="usenew"
```
Comment (ithqindia) [inline forms.blade.php]: `new` what is new? Add something meaningful like 'new_address'

File: forms.blade.php [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+        </form>
+    </div>
+</section>
+{{-- script --}}
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script>
+    $("[name='radiobtn']").click(function(event) {
+        if ($('#new').is(":checked")) {
+            $('#hide').show();
```
Comment (ithqindia) [inline forms.blade.php]: `#hide` is confusing word use something meanigful
